rust/noosphere-core/src/data/address.rs

```diff
@@ -99,15 +153,60 @@ impl LinkRecord {
                 }
             }
         }
-
         warn!("No facts contained a link!");
-
         None
     }
+
+    /// Return a base64-encoded string serialization of the
+    /// underlying [Ucan].
+    pub fn encode(&self) -> Result<String> {
```

Comment:
I would say just make this `Deref<Target = Ucan>` (which it already is, I think?) and invoke `encode` directly.

Alternatively, it this is important than we should eat our veggies and return a `Result<Jwt>` here.